subsys/logging/Kconfig

```diff
@@ -424,6 +424,51 @@ config LOG_BACKEND_RTT_FORCE_PRINTK
 
 endif # LOG_BACKEND_RTT
 
+config LOG_BACKEND_SPINEL
+	bool "Enable Spinel protocol backend"
```

Comment:
I'd mention here that this backend is dedicated to use with OpenThread, it might be not clear what Spinnel actually is. Here or in the help message.

Reply:
Done